src/shared/components/mutationTable/column/CategoricalColumnFormatter.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+
+export interface ICategoricalColumn {
+    displayValue?: string;
+    toolTip?: string;
+    className: string;
+}
+
+export default class CategoricalColumnFormatter {
+    /**
+     * Boilerplate code to create tool tip.
+     */
+    public static createToolTip(content: JSX.Element, toolTip: string) {

```

Comment:
We should probably make this a simple function and get rid of the `CategoricalColumnFormatter` class

```suggestion
    export function createToolTip(content: JSX.Element, toolTip: string) {
```

Reply:
thanks, @onursumer.  good idea.  this is now done.